File: 22. Hollow Right Triangle TL/htl.cpp
#include <iostream>
using namespace std;
int main(){
    int n;
    cout<<"Enter a Number : ";
    cin>>n;

    for(int i=1; i<=n; i++){
        for (int j=1; j<=i; j++){
            if(i==n or j==1 or i==j){
                cout<< "* ";
            }else{
                cout<< "  ";
            }
        }
        cout << '\n';
    }
}

/* Output

* 
* * 
*   * 
*     * 
* * * * * 

*/ 

File: Patterns/3. Right Triangle Patterns (Top-Left)/22. Hollow Right Triangle TL/htl.py
n = int(input("Enter a Number : "))

for i in range(1,n+1):
    for j in range(1,i+1):
        if(i==n or j==1 or i==j):
            print("*", end = ' ')
        else:
            print(end = '  ')
    print()

'''
Output:

* 
* * 
*   * 
*     * 
* * * * * 
 
'''